src/xmrEngine.js

```diff
@@ -797,8 +797,11 @@ class MoneroEngine {
         })
       )
     } catch (e) {
-      this.log.error(`makeSpend error: ${e}`)
-      throw e
+      const newError = new Error(e.message)
```

Comment:
Since you might receive multiple error messages, you might want to check which one you are getting to make sure you are parsing the correct error message.